projects/ui-policy-studio-angular/src/lib/components/step-form/gio-ps-step-form.component.ts

```diff
@@ -106,11 +106,11 @@ export class GioPolicyStudioStepFormComponent implements OnChanges, OnInit, OnDe
         this.infoBanner = this.genericPolicy.prerequisiteMessage;
       }
     }
-    if (changes.executionPhase) {
+    if (changes.flowPhase) {
       this.context = {
-        executionPhase: this.executionPhase,
+        flowPhase: this.flowPhase,
       };
-      this.isMessage = this.executionPhase === 'MESSAGE_REQUEST' || this.executionPhase === 'MESSAGE_RESPONSE';
+      this.isMessage = this.flowPhase === 'PUBLISH' || this.flowPhase === 'SUBSCRIBE';
```

Comment:
This would be true as well for Native APIs. Is that ok?

Reply:
yes. there's probably more. but that'll come later when i'm managing the phases for native. i'm aiming at the non regs with new inputs

Reply:
I'll note it to myself and see later ;)